src/tabmat/__init__.py

```diff
@@ -21,5 +21,8 @@
     "SplitMatrix",
     "CategoricalMatrix",
     "from_csc",
+    "from_formula",
     "from_pandas",
+    "as_tabmat",
+    "hstack",
```

Comment:
are we sure "hstack" is something we want to expose at the core of tabmat? I think it would be easier for people to import it using `from tabmat.split_matrix import hstack`

Reply:
It's a bit misleading that `hstack` lives in the `split_matrix` module, because it works on all `MatrixBase` subclasses, and returns the expected type for all combinations. I.e., its signature is
```
hstack(tup: Sequence[Union[MatrixBase, np.ndarray, sps.spmatrix]]) -> MatrixBase
```